styles/styles.scss

```diff
@@ -472,4 +472,198 @@ p {
       }
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
- [ ] [OPTIONAL] **GOOD JOB GUYS ⭐🙌👏**. I see you struggled with the spacing and background bug in the about section but don't worry I am here to help you 😄.

1. First remove the `margin-top: 150px;` from contact section in media query 992
2. Add this `padding: 9.6rem 3.2rem;` to `.about_bg-white_flex` inside 992px media query
It should already fix the problem but there are some issues in the mobile version also for this do these steps
3. Add this to the about section in CSS just to the mobile version, code, not inside a media query

All of these worked on my browser 😀👌

```css
    background: white;
    padding: 6.4rem 2.4rem;
```
All of these worked on my browser 😀👌

Reply:
Changes effected. Looks good! Thanks a lot!@Rustamxon7